### PR TITLE
OEA-385: Event Context Should be Stored and Displayed

### PR DESCRIPTION
**Pull request by marshl**
Changed events to have a firm_id column that is set to the context that the user selected when they created the event, and changed the diagnosis parameter to search on that relationship

### CODE REVIEW

**Comment by marshl** [comment]
On it

File: protected/views/patient/event_content.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	</header>
     <div class="event-content <?=($this->event->is_automated) ? 'auto' : ''?>" id="event-content">
 
-		<h2 class="event-title <?=($this->event->is_automated) ? 'auto' : ''?>" style="background-image: url('<?=$this->event->getEventIcon('medium')?>');"><?php echo $this->title?> <?php $this->renderPartial('//patient/event_automated'); ?></h2>
+		<h2 class="event-title <?=($this->event->is_automated) ? 'auto' : ''?>" style="background-image: url('<?=$this->event->getEventIcon('medium')?>');"><?php echo $this->title?> <?php echo $this->event->firm ? ' &mdash; Created under ' . $this->event->firm->name : null; ?> <?php $this->renderPartial('//patient/event_automated'); ?></h2>
```
Comment (apigram) [inline protected/views/patient/event_content.php]: You might have been able to use the PHP echo (<?=) shorthand here.

**Review by apigram — APPROVED** [review]

**Review by apigram — CHANGES_REQUESTED** [review]
The diagnosis case search parameter needs to be updated, along with its unit test.

**Comment by apigram** [comment]
Also, you'll likely need a migration to add the firm_id column.

**Comment by marshl** [comment]
@apigram <?= would be inconsistent in this case, so it can't be used. What do you mean the parameter needs to be updated?

**Review by apigram — APPROVED** [review]

**Pull request merged**